addons/payment/views/payment_templates.xml

```diff
@@ -315,6 +342,7 @@
         </form>
     </template>
 
+    
```

Comment:
Why? ^^

Reply:
That is good question, must have slipped my mind.